packages/hash/frontend/src/shared/layout/layout-with-header/search-bar.tsx

```diff
@@ -1,5 +1,5 @@
-import { useQuery } from "@apollo/client";
-import { PageSearchResult } from "@hashintel/hash-shared/graphql/apiTypes.gen";
+// import { useQuery } from "@apollo/client";
```

Comment:
Should we move this down to the commented out code (can always move it back up when uncommenting) or just delete it?